testhelpers/docker_registry.go

```diff
@@ -71,54 +72,61 @@ func NewDockerRegistry(ops ...RegistryOption) *DockerRegistry {
 func BasicAuth(handler http.Handler, username, password, realm string) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		user, pass, ok := r.BasicAuth()
-
 		if !ok || user != username || pass != password {
 			w.Header().Set("WWW-Authenticate", `Basic realm="`+realm+`"`)
 			w.WriteHeader(401)
 			_, _ = w.Write([]byte("Unauthorized.\n"))
 			return
 		}
-
 		handler.ServeHTTP(w, r)
 	})
 }
 
 // ReadOnly wraps a handler, allowing only GET and HEAD requests, otherwise rejecting with a 405
 func ReadOnly(handler http.Handler) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if !(r.Method == "GET" || r.Method == "HEAD") {
+	return http.HandlerFunc(func(w http.ResponseWriter, request *http.Request) {
```

Comment:
```suggestion
	return http.HandlerFunc(func(response http.ResponseWriter, request *http.Request) {
```

Reply:
Oh, I missed this one. Sorry it's fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1eb46031019fbbfccbc8f307cb6d467508e41a7d)